libfuse/mounter_osx.go

```diff
@@ -51,6 +54,7 @@ func GetPlatformSpecificMountOptionsForTest() []fuse.MountOption {
 	return []fuse.MountOption{
 		fuse.OSXFUSELocations(kbfusePath, fuse.OSXFUSELocationV3),
 		fuse.ExclCreate(),
+		// fuse.LocalVolume(),
```

Comment:
Please add a comment as to why this is disabled, and a TODO for getting it fixed.  Without the tests passing with this option enabled, I don't think we should consider KBFS-1751 finished.